.github/ISSUE_TEMPLATE/plugin-ownership-transfer.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Plugin Ownership Transfer
+about: Propose a plugin ownership transfer to the spinnaker-plugins organization. For more information, please see the documentation for [community donated plugins](https://github.com/spinnaker/governance/blob/master/sig-platform/community-donated-plugins.md)
+title: 'Plugin Transfer: <plugin name>'
+labels: committee/technical-oversight
+assignees: ajordens, ethanfrogers, ezimanyi, plumpy, robzienert
+
+---
+
+### Plugin Information
+
+Name of the plugin.
+
+Description of what the plugin is.
+
+### Acknowledgements
+
+- [ ] Source code will be available publicly on [`spinnaker-plugins`](https://github.com/spinnaker-plugins)
+- [ ] All authors listed below will be given `write` permissions to the plugin repository
```

Comment:
Does this structure mean that the author of one plugin has write access to all donated plugins?  Is that acceptable?  Is there any mechanism to restrict it to a particular plugin?

Should we describe what people or organizations should do when an author leaves the plugin project or donating organization?

Reply:
My understanding is that plugin "donors" would only be granted write access to *their* plugin's repository - is that correct?

Reply:
That would be my expectation. 

`spinnaker-plugins` is a new GitHub organization and each plugin is simply a repository within. Permissions would be managed per-repository.

Reply:
Correct - each plugin will have its own repository within the `spinnaker-plugins` org, and the permissions will be scoped to plugin repositories only. SC and TOC will have access to the org as admins, similar to the spinnaker org.

Reply:
Ack, thanks.

@pstout Does that sufficiently address your concern?

Reply:
This covers my question about the scope.  I don't think anybody has addressed my question about how we handle people moving on from working on a plugin or the organization that donated that plugin.  A sentence about who does what, or who is allowed to do what, would be useful down the road.

Reply:
Well, if I go look at the document, rather than just looking at this comment thread, I can see that you addressed the second part.  Sorry for the noise.